IFC4x3/PropertyEnumerations/t/PEnum_TransportEquipmentAsseblyType.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Should this PEnum be removed? Misspelled, but seems to overlap the correctly spelled PEnum below.

Reply:
This one has to be removed yes.